src/plugins/janus_audiobridge.c

```diff
@@ -8547,8 +8548,9 @@ static void *janus_audiobridge_participant_thread(void *data) {
 	janus_audiobridge_rtp_relay_packet *mixedpkt = NULL;
 	janus_rtp_header *rtp = NULL;
 	gint64 now = janus_get_monotonic_time(), before = now;
-	gboolean first = TRUE, use_fec = FALSE;
+	gboolean first = TRUE;
 	int ret = 0;
+	int lost_packets_gap = 0;
```

Comment:
I missed the logic for setting this to zero, you are right, thanks. I'll fix it.
Main idea of this - then webrtc client stops sending rtp media(e.g. in our case we are doing this then mic is muted on client), we still adding plc decodes and to avoid generation of it I added this condition. I could remove this logic, since this is not an expensive operation.

Reply:
fixed

Reply:
> Main idea of this - then webrtc client stops sending rtp media(e.g. in our case we are doing this then mic is muted on client), we still adding plc decodes and to avoid generation of it I added this condition. I could remove this logic, since this is not an expensive operation.
I see, that's a smart thing to do indeed.
In that case I'm not sure 20 packets are enough to consider incoming data as "silenced" (they are just about 400ms of audio, a network drop like that could not be so uncommon).
I'd consider increasing this number and test if generated concealed packets do not contain glitches of some kind that disrupts the audio or trigger the talking detection.